### ISSUE TITLE
[Debian] Hardly depends on libssl1.0.0 (Security Issue)

### ISSUE THREAD

**Issue by alsternerd** [haiwen/seafile-user-manual#19]
But needs to either depend on 1.0.2 which then needs updates, with every libssl update or implement it for 1.1.x or 1.x.

**Comment by killing** [comment]
What do you mean? For the client or for the server? It would be better to post on specific project.

**Comment by alsternerd** [comment]
Whereever it is used.

**Comment by openpaul** [comment]
It is currently impossible to install the seafile-gui from the debian 8 (jessy) repro via 
`apt-get install seafile-gui` which throws the error:

`The following packages have unmet dependencies:
 seafile-gui : Depends: libssl1.0.0 (>= 1.0.0) but it is not installable
               Depends: seafile-daemon (>= 5.1.2) but it is not going to be installed
`

Eventhough libssl1.0.2 is installed.

**Comment by FlorianBaumgartl** [comment]
Confirming @openpaul  error description with Debian 9 (Stretch).
```
The following packages have unmet dependencies:
seafile-gui : Depends: libssl1.0.0 (>= 1.0.0) but it is not installable
              Depends: ccnet (>= 5.1.2) but it is not going to be installed
              Depends: seafile-daemon (>= 5.1.2) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```
I'm using a fresh Debian 9.0 (Strech) installation:
```
Distributor ID:	Debian
Description:	Debian GNU/Linux 9.0 (stretch)
Release:	9.0
Codename:	stretch
   Linux tc-e73 4.9.0-2-amd64 #1 SMP Debian 4.9.18-1 (2017-03-30) x86_64 GNU/Linux
```

**Comment by killing** [comment]
It seems your repository information is quite old (5.1.x). Try updating to the latest version for Debian 8. That should also work on Stretch.

Another way is to configure testing branch on Stretch. The Debian developers are maintaining Seafile clients on testing branch.

**Comment by alsternerd** [comment]
Yes, that's right. It's April now and the ticket is from february.

**Comment by mrjovanovic** [comment]
Confirming, same issue using both the repo and the newly built packages from the Ubuntu PPA. Anyone have a workaround?
